backend/src/main/java/gov/cdc/usds/simplereport/db/model/PersonUtils.java

```diff
@@ -795,6 +795,11 @@ public static final Map<String, String> getResidenceTypeMap() {
   private static final String PREGNANT_SNOMED = "77386006";
   private static final String NOT_PREGNANT_SNOMED = "60001007";
   private static final String PREGNANT_UNKNOWN_SNOMED = "102874004";
+
+  public static final String YES_SYPHILIS_HISTORY_SNOMED = "1087151000119108";
```

Comment:
put a q out via Jayna for LAC to make sure these are the codes we should use. Will update these as necessary as that answer gets back to us.